proteus/NumericalFlux.py

```diff
@@ -761,7 +761,7 @@ def __init__(self,vt,getPointwiseBoundaryConditions,
         Advection_DiagonalUpwind_Diffusion_IIPG.__init__(self,vt,getPointwiseBoundaryConditions,
                                                          getAdvectiveFluxBoundaryConditions,
                                                          getDiffusiveFluxBoundaryConditions)
-        self.penalty_constant = 10.0
+        self.penalty_constant = 100.0
```

Comment:
Might be good to make a global constant again.

Reply:
On this one I'm going to just leave it as is. The best value of the penalty_constant in the DG-methods varies, unfortunately.